docs/src/main/asciidoc/container-image.adoc

```diff
@@ -9,16 +9,16 @@ include::./attributes.adoc[]
 
 Quarkus provides extensions for building (and pushing) container images. Currently it supports:
 
-- jib
-- docker
-- s2i
+- JIB
+- Docker
+- S2I
```

Comment:
Should we link to each extension's section?

Reply:
If we do, please use ids for the links.

Reply:
Is my change the correct way to do it?

Reply:
Not sure. Did you define ids on the paragraphs? I don’t see them in your diff but maybe they were preexisting (I’m on my phone).

Reply:
I didn't (I didn't even know you could). I'll update the PR in the morning

Reply:
Should be fixed with my latest change (I assume)